hack/test-cmd.sh

```diff
@@ -78,9 +84,15 @@ set -e
 
 function find_tests {
   find "${OS_ROOT}/test/cmd" -name '*.sh' -not -wholename '*images_tests.sh' | grep -E "${1}" | sort -u
```

Comment:
Rewrite this in a `for` loop (think you're not using Bash, but a normal language) and the case where you have no tests accumulated to print should be easy to detect right here (and `return` non zero)